src/utils.ts

```diff
@@ -238,8 +238,13 @@ export function addImportToCode (
 
   let _staticImports: StaticImport[] | undefined
   function findStaticImportsLazy () {
+    const original = s.original
+    const strippedCode = stripCommentsAndStrings(original)
```

Comment:
This looks not so efficient. Can we move it up to avoid bee executed repressively?